simply/scenario.py

```diff
@@ -146,15 +270,24 @@ def load(dirpath, data_format):
         at = actors_file.read_text()
         actors_j = json.loads(at)
         for aj in actors_j.values():
-            ai = [aj["id"], pd.read_csv(dirpath / aj["csv"]), aj["csv"], aj["ls"], aj["ps"],
-                  aj["pm"]]
+            ai = {"id": aj["id"],
+                  "df": pd.read_csv(dirpath / aj["csv"]),
+                  "csv": aj["csv"],
+                  "ls": aj["ls"],
+                  "ps": aj["ps"],
+                  "pm": aj["pm"]}
```

Comment:
only works for actor files but not market maker, maybe use a Class specific load function?

Reply:
the data is stored from a json dump anyways. therefore aj can be directly used in the constructor with just slight adjustment of csv data. No extra function is needed.